src/Illuminate/Database/Eloquent/Relations/HasOneOrMany.php

```diff
@@ -295,6 +320,27 @@ public function saveMany($models)
         return $models;
     }
 
+    /**
+     * Set the foreign key for a relationship to another model.
+     *
+     * @param Model $model
+     * @param string $relationship
+     * @return $this
+     */
```

Comment:
Please make sure your docblocs are in the same style as the rest of the framework (spaces & FQN).

Reply:
Apologies, I completely forgot to format them before I pushed up. I've just pushed an update to them, so they should hopefully be okay now? :)